### PR TITLE
stinky foxes test

### PR DESCRIPTION
**Pull request by angelheart04**
more will be added - a test for fox smell training patrols

### CODE REVIEW

**Comment by scribblecrumb** [comment]
Hey there!  Looks like you have some json issues.  Probably a trailing comma somewhere.  You'll have to resolve those before we can think about merging this one.

File: resources/dicts/patrols/mountainous/training/any.json [hunk 1]
```diff
@@ -2222,6 +2222,90 @@
...
+            {
+                "text": "app1 listens carefully as p_l explains how red foxes smell suspiciously like medicine cats - so much so, that red foxes are prone to attacking the medicine cats...",
+                "exp": 20,
+                "weight": 20
+            },
+            {
+                "text": "app1 picks up the instruction quickly and is soon able to recite how red foxes smell herbal, with kits smelling more like dandelion and adults like violets.",
+                "exp": 20,
+                "weight": 5
+            },
```
Comment (lillithrosepup) [inline resources/dicts/patrols/mountainous/training/any.json:2253]: Hey there! Remove the comma here ^.^

Comment (Thlumyn) [inline resources/dicts/patrols/mountainous/training/any.json:2253]:
```suggestion
            }
```
Remove this extra comma please

File: resources/dicts/patrols/mountainous/training/any.json [hunk 2]
```diff
@@ -2222,6 +2222,90 @@
...
+        "weight": 20,
+        "intro_text": "The patrol wants to hold a training session for app1. They decide to focus on how to identify different foxes.",
+        "decline_text": "They decide to focus on patrolling the territory instead. Training can wait for another day.",
+        "chance_of_success": 60,
+        "success_outcomes": [
+            {
+                "text": "app1 listens carefully as p_l explains how grey foxes roll in the dirtplace to disguise their scent.",
+                "exp": 20,
+                "weight": 20
+            },
```
Comment (Thlumyn) [inline resources/dicts/patrols/mountainous/training/any.json:2292]:
```suggestion
            }
```
Remove this extra comma, please

File: resources/dicts/patrols/mountainous/training/any.json [hunk 3]
```diff
@@ -2222,6 +2222,90 @@
...
+            {
+                "text": "s_c struggles to take the lesson seriously and eventually the training session ends early with no real progress made.",
+                "exp": 0,
+                "weight": 20,
+                "stat_trait": ["troublesome", "playful"],
+                "can_have_stat": ["app"]
+            }
+        ]
+    },
```
Comment (Thlumyn) [inline resources/dicts/patrols/mountainous/training/any.json:2308]: Lines 2308 and 2309 look like they can be removed, for a proper file end

**Review by Thlumyn — CHANGES_REQUESTED** [review]
Please test your PRs; this code does not run as it is, the json file that has been modified is not a valid json

**Comment by Thlumyn** [comment]
Closing as stale; please reopen once those json fixes are made

**Pull request closed without merging**